Bumps cyclonedx-maven-plugin to version 2.8.0 and tweaks its configuration

From: dervoeti <lukas.voetmand@stackable.tech>


---
 pom.xml |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/pom.xml b/pom.xml
index 8bd80e81e2..a437d5c819 100755
--- a/pom.xml
+++ b/pom.xml
@@ -3011,7 +3011,11 @@
           <plugin>
             <groupId>org.cyclonedx</groupId>
             <artifactId>cyclonedx-maven-plugin</artifactId>
-            <version>2.7.10</version>
+            <version>2.8.0</version>
+            <configuration>
+                <projectType>application</projectType>
+                <schemaVersion>1.5</schemaVersion>
+            </configuration>
             <executions>
               <execution>
                 <goals>
